site/survey.html

```diff
@@ -0,0 +1,124 @@
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+    <link rel="manifest" href="/site.webmanifest" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico">
+</head>
+<body>
+    <div class="width-wrap">
+        <header id="top">
+            <nav><ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="links.html">Links</a></li>
+                <li><a href="glossary.html">Glossary</a></li>
+            </ul></nav>
+        </header>
+        <main>
+            <div id="survey">
+                <h1>Survey</h1>
+                <form action="https://formspree.io/f/xvgrrwyd" method="post">
+                    <section>
+                        <p>
+                            <label for="name">
+                                <span>What is your name? </span>
+                                <strong><span aria-label="required">*</span></strong>
+                            </label>
+                            <input type="text" id="name" name="name_field" required />
+                        </p>
+                        <p>
+                            <label for="email">
+                                <span>What is your email? </span>
+                                <strong><span aria-label="required">*</span></strong>
+                            </label>
+                            <input type="text" id="email" name="email_field" required />
+                        </p>
+                        <p>
+                            <label for="whole_number">
+                                <span>Which whole number is best? </span>
+                            </label>
+                            <input type="number" min="0" step="1" id="whole_number" name="best_whole_number" />
+                        </p>
+                        <p>
+                            <label for="date">
+                                <span>Which day is best? </span>
+                            </label>
+                            <input type="date" id="date" name="best_day" />
+                        </p>
+                        <p>
+                            <label for="bear">
+                                <span>Which bear is best? </span>
+                                <strong><span aria-label="required">*</span></strong>
+                            </label>
+                            <input list="bear" name="best_bear" required />
+                            <datalist id="bear">
+                                <option value="" selected>&nbsp;</option>
+                                <option>Black</option>
+                                <option>Brown</option>
+                                <option>Care</option>
+                                <option>Panda</option>
+                                <option>Polar</option>
+                                <option>Teddy</option>
+                            </datalist>
```

Comment:
Not necessarily wrong but expected something akin to:
```suggestion
<label for="best_bear">Which bear is best? <abbr title="required" aria-label="required">*</abbr></label>
          <select id="best_bear" name="best_bear" required aria-required="true">
            <option selected value="">&nbsp;</option>
            <option value="Black">Black</option>
            <option value="Brown">Brown</option>
            <option value="Care">Care</option>
            <option value="Panda">Panda</option>
            <option value="Polar">Polar</option>
            <option value="Teddy">Teddy</option>
          </select>
```